testutil/keeper/monitoring_provider.go

```diff
@@ -57,7 +57,7 @@ func NewTestSetupWithIBCMocksMonitoringp(
 	profileKeeper := initializer.Profile()
 	launchKeeper := initializer.Launch(profileKeeper, distrKeeper, paramKeeper)
 	rewardKeeper := initializer.Reward(authKeeper, bankKeeper, profileKeeper, launchKeeper, paramKeeper)
-	projectKeeper := initializer.Project(launchKeeper, profileKeeper, bankKeeper, distrKeeper, *rewardKeeper, paramKeeper, fundraisingKeeper)
+	projectKeeper := initializer.Project(launchKeeper, profileKeeper, bankKeeper, distrKeeper, paramKeeper)
```

Comment:
@Pantani do you think we should create an issue to migrate SPN modules [away](https://github.com/cosmos/cosmos-sdk/blob/release/v0.47.x/UPGRADING.md#xparams) from `x/params` in preparation for Cosmos SDK `v0.48`?

Reply:
yes! good idea!!

Reply:
https://github.com/tendermint/spn/issues/1012